src/Command/Wishlist/AddWishlistProductInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Command\Wishlist;
+
+use BitBag\SyliusWishlistPlugin\Entity\WishlistProductInterface;
+use Sylius\Bundle\OrderBundle\Controller\AddToCartCommandInterface;
+
+interface AddWishlistProductInterface
```

Comment:
It's a bit unclear to me, what it does: does it add a wishlist product to cart or creates a wishlist item from product?